services/management/checks.go

```diff
@@ -126,6 +133,17 @@ func (s *ChecksAPIService) ChangeSecurityChecks(ctx context.Context, req *manage
 		}
 	}
 
+	if len(changeIntervalParams) != 0 {
+		err := s.checksService.ChangeInterval(changeIntervalParams)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to change security check interval")
+		}
+		// enabling/disabling checks and changing interval are two seperate
+		// features on the UI that shrare the same API endpoint.
+		// if check interval was changed without error then return early.
+		return &managementpb.ChangeSecurityChecksResponse{}, nil
```

Comment:
But why? Why user can enable check and set interval simultaneously?

Reply:
i explained it in the comment, according to the UI design these were two different things
but on second thoughts maybe it will be better to remove that so that we don't have to change the backend in case the UI allows users to do those two things together in the future

Reply:
done